### PR TITLE
DOCSP-12575: kms azure python updates

### PR DESCRIPTION
**Pull request by ccho-mongodb**
JIRA:
https://jira.mongodb.org/browse/DOCSP-12575
https://jira.mongodb.org/browse/DOCSP-13057 (broken out)

No assets to stage

### CODE REVIEW

File: python/app.py [hunk 1]
```diff
@@ -17,30 +17,58 @@
...
+    For GCP KMS, uncomment this block. See https://docs.mongodb.com/drivers/security/client-side-field-level-encryption-local-key-to-kms
+    for more information on storing a master key on a KMS.
+    """
 
-    schema = csfle_helper.create_json_schema(data_key=data_key)
-    csfle_enabled_client = csfle_helper.get_csfle_enabled_client(schema)
-    regular_client = csfle_helper.get_regular_client()
+    keyDb = "encryption"
+    keyColl = "__keyVault"
+    keyNamespace = f"{keyDb}.{keyColl}"
```
Comment (terakilobyte) [inline python/app.py:68]: this ends up not being used

**Review by terakilobyte — CHANGES_REQUESTED** [review]
small things I noticed

File: python/app.py [hunk 2]
```diff
@@ -17,30 +17,58 @@
...
-            "key": local_master_key,
+            "key": master_key,
         },
     }
 
-    csfle_helper = CsfleHelper(kms_providers=kms_providers)
+    """
+    For AWS KMS, uncomment this block. See https://docs.mongodb.com/drivers/security/client-side-field-level-encryption-local-key-to-kms
+    for more information on storing a master key on a KMS.
+    """
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply python/app.py:39]: Hm, I think this was intentional, but is missing the actual AWS KMS code. Will fill in.

**Review by terakilobyte — APPROVED** [review]
lgtm

**Pull request merged**